lib/quill_delta.dart

```diff
@@ -283,7 +283,7 @@ class Delta {
   }
 
   /// Returns list of operations in this delta.
-  List<Operation> toList() => List.from(_operations);
+  List<Operation> toList() => _operations; // List.from(_operations);
```

Comment:
already covered by operationsUnsafe, no?

```suggestion
  List<Operation> toList() => List.from(_operations);
```

Reply:
That was committed by accident.